diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/FileUtils.kt

```diff
@@ -7,14 +7,15 @@ package org.cqfn.diktat.ruleset.utils
 internal const val SRC_DIRECTORY_NAME = "src"
 
 /**
- * Splits [this] string by file path separator
+ * Splits [this] string by file path separator. Drop last element - file type
```

Comment:
Please update kdoc correctly. Also, file can have more than one dots. You should drop last after splitting by directory separator and only then split remaining tokens by dot

Reply:
Fixed